doc/user/pim.rst

```diff
@@ -434,200 +382,167 @@ cause great confusion.
 
-.. index:: show ip pim assert
 .. clicmd:: show ip pim assert
 
    Display information about asserts in the PIM system for S,G mroutes.
 
-.. index:: show ip pim assert-internal
 .. clicmd:: show ip pim assert-internal
 
    Display internal assert state for S,G mroutes
 
-.. index:: show ip pim assert-metric
 .. clicmd:: show ip pim assert-metric
 
    Display metric information about assert state for S,G mroutes
 
-.. index:: show ip pim assert-winner-metric
 .. clicmd:: show ip pim assert-winner-metric
 
    Display winner metric for assert state for S,G mroutes
 
-.. index:: show ip pim group-type
 .. clicmd:: show ip pim group-type
 
    Display SSM group ranges.
 
-.. index:: show ip pim interface
 .. clicmd:: show ip pim interface
 
    Display information about interfaces PIM is using.
 
-.. index:: show ip pim mlag [vrf NAME] interface [detail|WORD] [json]
 .. clicmd:: show ip pim mlag [vrf NAME|all] interface [detail|WORD] [json]
 
    Display mlag interface information.
 
-.. index:: show ip pim [vrf NAME] join [A.B.C.D [A.B.C.D]] [json]
 .. clicmd:: show ip pim join
 
    Display information about PIM joins received.  If one address is specified
    then we assume it is the Group we are interested in displaying data on.
    If the second address is specified then it is Source Group.
 
-.. index:: show ip pim local-membership
 .. clicmd:: show ip pim local-membership
 
    Display information about PIM interface local-membership.
 
-.. index:: show ip pim mlag summary [json]
 .. clicmd:: show ip pim mlag summary [json]
 
    Display mlag information state that PIM is keeping track of.
 
-.. index:: show ip pim neighbor
 .. clicmd:: show ip pim neighbor
 
    Display information about PIM neighbors.
 
-.. index:: show ip pim nexthop
 .. clicmd:: show ip pim nexthop
 
    Display information about pim nexthops that are being used.
 
-.. index:: show ip pim nexthop-lookup
 .. clicmd:: show ip pim nexthop-lookup
 
    Display information about a S,G pair and how the RPF would be chosen. This
    is especially useful if there are ECMP's available from the RPF lookup.
 
-.. index:: show ip pim rp-info
 .. clicmd:: show ip pim rp-info
 
    Display information about RP's that are configured on this router.
 
-.. index:: show ip pim rpf
 .. clicmd:: show ip pim rpf
 
    Display information about currently being used S,G's and their RPF lookup
    information. Additionally display some statistics about what has been
    happening on the router.
 
-.. index:: show ip pim secondary
 .. clicmd:: show ip pim secondary
 
    Display information about an interface and all the secondary addresses
    associated with it.
 
-.. index:: show ip pim state
 .. clicmd:: show ip pim state
 
    Display information about known S,G's and incoming interface as well as the
    OIL and how they were chosen.
 
-.. index:: show ip pim [vrf NAME] upstream [A.B.C.D [A.B.C.D]] [json]
 .. clicmd:: show ip pim upstream
```

Comment:
losing the options?

Reply:
Should have been in the clicmd definition but I'll fix this one